text/machine_comprehension/bert-squad/README.md

```diff
@@ -0,0 +1,111 @@
+# BERT-Squad
+
+## Use cases
+This model uses masked language method to enable training of bidirectional models that can quickly and efficiently run through language processing tasks. The model also adds a next sentence prediction task to improve sentence-level understanding which helps to answer questions based on the given paragraph input. 
```

Comment:
This sounds like how the model is trained. Use case should describe why someone would use this model, what scenario is addresses (for example, for resnet, it's classifying images into one of 1000 categories.)